tests/test_acceptance.py

```diff
@@ -173,9 +209,9 @@ class Meta:
             float_field = FloatField(name="floatField", required=True, allow_null=False)
             boolean_field = BooleanField(name="boolField", required=True, allow_null=False)
             datetime_field = DateTimeField(name="datetimeField", required=True, allow_null=False)
-            date_field = DateField(name="dateField", required=True, allow_null=False)
-            time_field = TimeField(name="timeField", required=True, allow_null=False)
-            uuid_field = UuidField(name="uuidField", required=True, allow_null=False)
+            date_field = DateField(name="dateField", required=True, allow_null=True)
```

Comment:
Part of this test is ensuring that these the `allow_null=False` setting is being enforced.  It probably shouldn't be changed in an acceptance test here.